Include/cpython/unicodeobject.h

```diff
@@ -442,18 +442,18 @@ PyAPI_FUNC(PyObject*) PyUnicode_FromKindAndData(
 
 /* --- Manage the default encoding ---------------------------------------- */
 
-/* Returns a pointer to the default encoding (UTF-8) of the
-   Unicode object unicode.
-
-   Like PyUnicode_AsUTF8AndSize(), this also caches the UTF-8 representation
-   in the unicodeobject.
-
-   Use of this API is DEPRECATED since no size information can be
-   extracted from the returned data.
-*/
-
+// Returns a pointer to the default encoding (UTF-8) of the
```

Comment:
Was it necessary to change the type of the comment? Most comments here are `/* */`.

It makes reviewing the changes more difficult.

Reply:
It's not necessary, but I take this PR as an opportunity to change the comment style. IMO multi-line comments written with ``// comment`` syntax are way easier to read.